Soccar [Physics]/src/nl/soccar/physics/GameEngine.java

```diff
@@ -176,6 +188,14 @@ public void removeWorldObject(WorldObject object) {
         }
     }
 
+    public void resetWorldObjects() {

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)